openssl.yaml

```
@@ -165,6 +167,12 @@ advisories:
     - timestamp: 2023-03-28T07:54:27.093515-07:00
       status: fixed
       fixed-version: 3.1.0-r2
+  CVE-2023-0466:
+    - timestamp: 2023-04-08T12:32:54.797413-04:00
+      status: not_affected
+      justification: component_not_present
+      impact: The upstream behavior was deemed correct, and the documentation was updated instead of a code change
```

IIUC, something like this (feel free to tweak) would be more accurate in this case:

```suggestion
      impact: The upstream maintainers decided to document a workaround rather than make a code change that would potentially break existing users.
```

---

Nice work 👏